### PR TITLE
generalize camera assumptions

### PR DESCRIPTION
**Pull request by ZwX1616**

### CODE REVIEW

File: common/transformations/camera.py [hunk 1]
```diff
@@ -1,29 +1,67 @@
...
+  [0.0,  eon_f_focal_length,  float(eon_f_frame_size[1])/2],
+  [0.0,  0.0,                                          1.0]])
+eon_intrinsics = eon_fcam_intrinsics # xx
 
 leon_dcam_intrinsics = np.array([
-  [650,   0,   816//2],
-  [  0,  650,  612//2],
-  [  0,    0,     1]])
+  [leon_d_focal_length,  0.0,  float(leon_d_frame_size[0])/2],
+  [0.0,  leon_d_focal_length,  float(leon_d_frame_size[1])/2],
```
Comment (HaraldSchafer) [inline common/transformations/camera.py:27]: You changed this array from int to float. If that was unintentional make sure float is what we want. I see the eon_intrinsics are float too.

Comment (ZwX1616) [reply common/transformations/camera.py:27]: I think the camera intrinsics are mostly used with other float quantities such as calib.
And yes the eon_intrinsics has been float.

File: common/transformations/camera.py [hunk 2]
```diff
@@ -137,18 +175,18 @@ def img_from_device(pt_device):
...
-  calib_frame_from_ground = np.dot(eon_intrinsics,
+  calib_frame_from_ground = np.dot(intrinsics,
                                      get_view_frame_from_road_frame(0, 0, 0, 1.22))[:, (0, 1, 3)]
   ground_from_calib_frame = np.linalg.inv(calib_frame_from_ground)
   camera_frame_from_calib_frame = np.dot(camera_frame_from_ground, ground_from_calib_frame)
   return camera_frame_from_calib_frame
 
 
-def pretransform_from_calib(calib):
+def pretransform_from_calib(calib, intrinsics=fcam_intrinsics):
```
Comment (HaraldSchafer) [inline common/transformations/camera.py:187]: this function is unused, can be removed

**Comment by HaraldSchafer** [comment]
Besides that LGTM

**Pull request merged**